code/api/package.json

```diff
@@ -1,49 +1,49 @@
 {
-  "name": "api",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "setup": "npm run setup:install && npm run setup:db",
-    "setup:install": "npm install",
-    "setup:db": "npm run db:migrate && npm run db:seed",
-    "db:migrate": "./node_modules/.bin/sequelize db:migrate",
-    "db:seed": "./node_modules/.bin/sequelize db:seed:all",
-    "start": "nodemon src/index.js --exec babel-node",
-    "start:prod": "npm run build:prod && npm run start:server",
-    "build:prod": "babel src -s -D -d build",
-    "start:server": "node build/index.js",
-    "test": "jest --watch" 
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "devDependencies": {
-    "@babel/cli": "7.8.4",
-    "@babel/core": "7.9.0",
-    "@babel/node": "7.8.7",
-    "@babel/plugin-transform-runtime": "7.9.0",
-    "@babel/preset-env": "7.9.5",
-    "jest": "^26.4.1",
-    "nodemon": "2.0.3",
-    "supertest": "^4.0.2"
-  },
-  "dependencies": {
-    "@babel/runtime": "7.9.2",
-    "bcrypt": "4.0.1",
-    "body-parser": "1.19.0",
-    "cookie-parser": "1.4.5",
-    "cors": "2.8.5",
-    "dotenv": "8.2.0",
-    "express": "4.17.1",
-    "express-graphql": "0.9.0",
-    "graphql": "14.6.0",
-    "jsonwebtoken": "8.5.1",
-    "morgan": "1.10.0",
-    "multer": "1.4.2",
-    "mysql2": "2.1.0",
-    "pg": "^8.2.1",
-    "sequelize": "5.21.6",
-    "sequelize-cli": "^5.5.1"
-  }
+    "name": "api",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "setup": "npm run setup:install && npm run setup:db",
+        "setup:install": "npm install",
+        "setup:db": "npm run db:migrate && npm run db:seed",
+        "db:migrate": "./node_modules/.bin/sequelize db:migrate",
+        "db:seed": "./node_modules/.bin/sequelize db:seed:all",
+        "start": "nodemon src/index.js --exec babel-node",
+        "start:prod": "npm run build:prod && npm run start:server",
+        "build:prod": "babel src -s -D -d build",
+        "start:server": "node build/index.js",
+        "test": "jest --watch"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "devDependencies": {
+        "@babel/cli": "7.8.4",
+        "@babel/core": "7.9.0",
+        "@babel/node": "7.8.7",
+        "@babel/plugin-transform-runtime": "7.9.0",
+        "@babel/preset-env": "7.9.5",
+        "jest": "^26.4.1",
+        "nodemon": "2.0.3",
+        "supertest": "^4.0.2"
+    },
+    "dependencies": {
+        "@babel/runtime": "7.9.2",
+        "bcrypt": "4.0.1",
+        "body-parser": "1.19.0",
+        "cookie-parser": "1.4.5",
+        "cors": "2.8.5",
+        "dotenv": "8.2.0",
+        "express": "4.17.1",
+        "express-graphql": "0.9.0",
+        "graphql": "14.6.0",
+        "jsonwebtoken": "8.5.1",
+        "morgan": "1.10.0",
+        "multer": "1.4.2",
+        "mysql2": "2.1.0",
+        "pg": "^8.2.1",
+        "sequelize": "5.21.6",
+        "sequelize-cli": "^5.5.1"
+    }
```

Comment:
so uh, did anything change here aside from the indentations?

Reply:
no, sorry, I have setting for 2 now, but it doesn't change it back